tests/integration/golang/helpers/env.go

```diff
@@ -26,6 +26,14 @@ func GetS3EndpointUri() string {
 	return "http://localhost:9000"
 }
 
+func GetGSEndpointUri() string {
+	uri, ok := os.LookupEnv("FML_GS_ENDPOINT_URI")
+	if ok {
+		return uri
+	}
+	return "http://localhost:4333"
```

Comment:
this is nice but we then need to have the fake GCS server brought up with the devcontainer

Reply:
done